components/Navigation.tsx

```diff
@@ -134,10 +134,11 @@ const navigation: navigationItem[] = [
 
       {
         type: 'route',
-        title: 'Connection Setup',
-        path: '/data-design/define-sources-and-destinations',
```

Comment:
Should we worry about re-directing from here to the new place or just let it slide?